app/views/pages/tools.html.erb

```diff
@@ -38,7 +38,7 @@
       </li>
       <li>
         <p>
-          <%= link_to t('breadcrumb.footnotes'), footnote_search_path %>
+          <%= link_to t('breadcrumb.footnote_search'), footnote_search_path %>
```

Comment:
This localisation was broken and only noticed because I changed the test localisation config to raise on misses